browser/src/slideshow/LayerDrawing.ts

```diff
@@ -621,6 +621,10 @@ class LayerDrawing {
 			this.prefetchedSlideHash = null;
 			return;
 		}
+
+		if (!this.requestedSlideHash)
+			return;
```

Comment:
in case better to log a warning on the console

Reply:
Ok, adding warning message